tests/README.md

```diff
@@ -1,26 +1,52 @@
 ## TDX Tests
 
-The tests should be run locally on TDX host.
+This folder contains TDX tests.
 
-It is assumed that these commands are running on a TDX host
-that has been setup properly.
+### Pre-requisites
 
-The guest image must be available at the location : `checkbox-provider-tdx/data/tdx-guest.qcow2`
-The guest image must enable ssh server with password-based authentication for `root` user.
-The root user password must be `123456`
+- The tests must be executed a TDX host that has been setup properly.
 
-Run tests with pytest
+- The guest image must be available at the project folder with the name `tdx-guest.qcow2`
+  You can follow the instructions in the project README to create one guest image.
+  You can specify a path to the guest image with `TDXTEST_IMAGE_FILE` environment variable.
```

Comment:
I suggest adding something here like: "If you do export this environment variable, be sure to run the `sudo` commands below with the -E option to preserve environment variables."

Reply:
I think we should let the choice to users to do it the way they want, for example, they can chose to do : `sudo TDXTEST_IMAGE_FILE=xxx run.sh pytest` .....

Reply:
Yeah, that's reasonable. Sometimes I spoon feed the user too much. :)